keyboards/melgeek/z70ultra_v1/rev1/rev1.c

```diff
@@ -0,0 +1,167 @@
+    {0, CS6_SW4, CS8_SW4, CS7_SW4},    /* RGB67 */
+    {0, CS6_SW5, CS8_SW5, CS7_SW5},    /* RGB69 */
+    {0, CS6_SW6, CS8_SW6, CS7_SW6},    /* RGB70 */
+    {0, CS6_SW2, CS8_SW2, CS7_SW2},    /* RGB65 */
+
+    {0, CS37_SW1, CS39_SW1, CS38_SW1}, /* RGB19 */
+    {0, CS37_SW2, CS39_SW2, CS38_SW2}, /* RGB20 */
+    {0, CS37_SW4, CS39_SW4, CS38_SW4}, /* RGB22 */
+    {0, CS37_SW3, CS39_SW3, CS38_SW3}, /* RGB21 */
+    {0, CS31_SW9, CS33_SW9, CS32_SW9}, /* RGB36 */
+};
+
+led_config_t g_led_config = {
+	{
+		/* C0     C1      C2      C3      C4      C5      C6      C7      C8      C9       C10      C11     C12     C13     C14 */
+		{  0,      1,      2,      3,     4,       5,      6,      7,      8,      9,       10,      11,     12,     13,     14, }, /* R0 */
+		{ 15,     16,     17,     18,     19,     20,     21,     22,     23,     24,       25,      26,     27,     28,     29, }, /* R1 */
+		{ 30,     31,     32,     33,     34,     35,     36,     37,     38,     39,       40,      41,     NO_LED, 42,     43, }, /* R2 */
+		{ 44,     NO_LED, 45,     46,     47,     48,     49,     50,     51,     52,       53,      54,     55,     56,     57, }, /* R3 */
+		{ 58,     59,     60,     NO_LED, 61,     62,     63,     NO_LED, NO_LED, 64,       65,      NO_LED, 66,     67,     68, }, /* R4 */
+    }, {
+        {0,  0},  {16,  0},  {32,  0}, {48,  0}, {64,  0}, {80,  0}, {96,  0}, {112,  0}, {128,  0}, {144,  0}, {160,  0}, {176,  0}, {192,  0}, {208,  0}, {224,  0},
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16}, {224, 16},
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32},            {192, 32}, {208, 32},
+		{0, 48},             {16, 48}, {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48},  {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48}, {192, 48}, {208, 48},
+		{0, 64},  {16, 64},  {32, 64},           {48, 64}, {64, 64}, {80, 64},                       {96, 64},  {112, 64},            {144, 64}, {160, 64}, {176, 64},
+	}, {
+	       4,       4,         4,        4,        4,        4,        4,        4,          4,         4,         4,          4,        4,         4,        4,
+		   4,       4,         4,        4,        4,        4,        4,        4,          4,         4,         4,          4,        4,         4,        4,
+		   4,       4,         4,        4,        4,        4,        4,        4,          4,         4,         4,          4,                   4,        4, 
+		   4,                  4,        4,        4,        4,        4,        4,          4,         4,         4,          4,        4,         4,        4, 
+		   4,       4,         4,                  4,        4,        4,                               4,         4,                    4,         4,        4, 
+	}
+};
+
+const is31_led g_is31_indicator_leds[6] = {
```

Comment:
Managing these as a separate array doesnt seem the cleanest solution over `rgb_matrix_indicators_kb` like the majority of other keyboards.